src/field_ext.rs

```diff
@@ -30,10 +30,18 @@ impl FieldExt for Fq {
         debug_assert_eq!(self.serialized_size(), 32);
         self.serialize(&mut bytes[..])
             .expect("serialization into array should be infallible");
+        // Set top three bits of last byte to zero
+        bytes[31] &= 0b00011111;
```

Comment:
Hmm, if the top three bits of the serialization were nonzero, something's gone badly wrong, right? Should this be a `debug_assert` instead?